packages/ia-pic-uploader/src/ia-pic-uploader.ts

```diff
@@ -1,10 +1,11 @@
-import { html, css, LitElement, CSSResultGroup, nothing, svg } from 'lit';
-import { property, customElement, state, query } from 'lit/decorators.js';
-import iaButtonStyle from './style/ia-button-style';
+import { css, CSSResultGroup, html, LitElement, nothing, svg } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+
+import '@internetarchive/ia-activity-indicator';
```

Comment:
I hope you have verified by QA on your local system to see loading render correct.

Reply:
It does:

<img width="463" alt="Screenshot 2025-03-11 at 19 12 03" src="https://github.com/user-attachments/assets/9622cab4-41ad-4f8a-96c5-38013ca779f7" />